src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them. Refer to the checklist item 'Don't use * selector (it impacts performance)'.